### DISCUSSION TITLE
软件图标看起来有点糊，希望漫画和小说阅读页面可以状态栏沉浸

### DISCUSSION THREAD

**Discussion by qa12587** [xiaoyaocz/flutter_dmzj#117] [General]
。

**Comment by xiaoyaocz** [comment]
现在状态栏不是沉浸的吗

**Reply by qa12587** [reply]
并没有，指的是漫画小说阅读页面。至少我小米10没有
![Uploading Screenshot_2023-03-16-21-51-06-756_com.xycz.dmzjx.jpg…]()

**Reply by xiaoyaocz** [reply]
你的图挂了，现在这样不是沉浸的吗
![20230317090049.jpg](https://vip2.loli.io/2023/03/17/cl8kqV4S5hCxWsm.jpg)
![20230317090055.jpg](https://vip2.loli.io/2023/03/17/NM3RczBlELVhYiD.jpg)

**Reply by qa12587** [reply]
应该是这种说，因为挖空屏导致阅读页面不能全覆盖留下一部分黑色

**Comment by qa12587** [comment]
![Screenshot_2023-03-17-19-29-37-971_com xycz dmzjx](https://user-images.githubusercontent.com/116661799/225892822-90e2ed7c-11f5-4a69-bb3a-ec3d4a1d16f1.jpg)

**Comment by xiaoyaocz** [comment]
#143
